packages/portal-proto/src/components/Summary/CategoryTableSummary.tsx

```diff
@@ -30,13 +30,13 @@ function CategoryTableSummary<TData>({
             withArrow
             withinPortal={false}
           >
-            <ActionIcon variant="subtle">
+            <span className="flex items-center px-1.5">
```

Comment:
This is an a11y fix. ActionIcon wraps it in a button which needs a name/label (and button is not the best semantic option for this element).